workflow/scripts/compute_diversity_measures.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
```

Comment:
This file should be formatted with `black`.

Reply:
Done. Thanks 👍